docs/src/further_topics/ugrid/operations.rst

```diff
@@ -60,8 +60,26 @@ subsequent example operations on this page.
 
         >>> import numpy as np
 
+                >>> from iris.coords import AuxCoord
+                >>> from iris.experimental.ugrid import Connectivity, MeshXY
+
+                # Going to create the following mesh
+                #  (node indices are shown to aid understanding):
+                #
+                #  0----1
+                #  |    |
+
+                >>> from iris.coords import AuxCoord
+                >>> from iris.experimental.ugrid import Connectivity, MeshXY
+
+                # Going to create the following mesh
+                #  (node indices are shown to aid understanding):
+                #
+                #  0----1
+                #  |    |
+
```

Comment:
Hey @trexfeathers, this appears to be here by mistake. Could you remove it please?

Reply:
ea7821743d6fe8ed4cbccc729cdadc1ecea68880